helpers/combine_docfiles.py

```diff
@@ -50,6 +50,8 @@
 replace_content = groups[0] + groups[1]
 
 # Find where to put the replacement content, overwrite the input file
+# TODO(jmccune) fix AttributeError: 'NoneType' object has no attribute 'groups'
```

Comment:
We should open issues rather than adding TODOs, if they aren't reasonable to address in the existing pull request.

Reply:
Removed this TODO and opened a bug.